contents/handbook/growth/customer-support.md

```diff
@@ -22,6 +22,49 @@ These are the ways in which customers can currently reach us:
 
 > Sometimes users ask about the progress of [certain issues](https://github.com/PostHog/posthog) that are important to them on GitHub. We don't consider GitHub to be a proper 'support' channel, but it is a useful place to gauge the popularity of feature requests or the prevalence of issues. 
 
+### Response Targets
+
+We have a high volume of tickets created, and some require more attention than others.  Here we have our targets for _initial_ response/acknowledgement to issues by Customer Priority and Issue [Severity](/docs/support-options#severity-levels), along with an indication of who is responsible for the initial response (CS = Customer Success, SH = designated Support Hero for the impacted team)
+
+<table>
+  <tr>
+    <th colspan="2"></th><th colspan="3"> Customer Priority</th>
+  </tr>
+  <tr>
+    <td colspan="2"></td>
+    <th>High</th>
+    <th>Medium</th>
+    <th>Low</th>
+  </tr>
+<tr>
+    <th rowspan="4">Issue Severity</th>
+    <th>Critical</th>
+    <td>1 hour (CS)</td>
+    <td>1 hour (CS)</td>
+    <td>1 day (CS)</td>
+  </tr>
+<tr>
+    <th>High</th>
+    <td>1 hour (SH)</td>
```

Comment:
Is it feasible to expect an engineering support hero to get back to a high priority customer with a high severity issue in an hour? Agree if it's Marcus, but otherwise they could be spending that time fixing the issue potentially?

Reply:
That one was a typo - I've pushed a fix.  Thanks for flagging!